src/containers/App/App.js

```diff
@@ -40,9 +42,10 @@ const App = (props) => {
   const hasP2PSocket =
     process.env.REACT_APP_P2P_RIPPLED_HOST != null &&
     process.env.REACT_APP_P2P_RIPPLED_HOST !== ''
+  const p2pConnection = process.env.REACT_APP_P2P_UNENCRYPTED ?? 'wss'
```

Comment:
I dont think this is needed since we now have a clio for the nft devnet setup with wss.

Reply:
which clio would that be? im not sure if it supports nft_info